policyengine_us/variables/gov/states/co/ssa/oap/co_oap.py

```diff
@@ -10,8 +10,11 @@ class co_oap(Variable):
 
     def formula(person, period, parameters):
         income = person("ssi_countable_income", period)
+        is_joint = person("ssi_claim_is_joint", period)
         ssi = person("ssi", period)
-        total_countable_income = ssi + income
+        # The SSI variable assigns SSI to one member of the marital unit if both are eligible
+        marital_ssi = where(is_joint, person.marital_unit.sum(ssi) / 2, ssi)
```

Comment:
```suggestion
        marital_ssi = where(is_joint, person.household.sum(ssi) / 2, ssi)
```
I think we want to sum it by household

Reply:
SSI is a per member benefit except for the marital unit. So, if the child qualifies for SSI, we would not want that to be counted.

Reply:
Could you add a source for this? AFAIUI, spouses could get different portions of the marital unit's total SSI, based on the personal income of each spouse; it's not automatically split 50/50.

Reply:
My understanding is that if both spouses are eligible for SSI, they would only receive one check.

FYI, I just realized that I used the wrong variable. Just changed it to use `ssi_marital_both_eligible`

Reply:
Could you share a source on that?

Reply:
What part of that? This seems related but doesn't back this up AFAICT:
>Deduct an amount to meet the needs of each dependent child living in the household of
the spouse equal to half the maximum SSI grant benefit standard

Could you please provide a full justification for this change in #5275? We prefer to document rationale in issues especially for nontrivial changes here. I'm not yet convinced so please include legal sources and pointers to our current code that needs correction.

Reply:
@MaxGhenis Sorry for the delay on this; we had to do some more research. I was wrong about couples receiving one check. According to the [handbook](https://www.ssa.gov/OP_Home/handbook/handbook.01/handbook-0131.html), couples receive separate checks. We also compiled some more detailed notes [here](https://docs.google.com/document/d/1D-pKsKqNIRhu7ksXIcAXMHBH9aX-g_-Nz9dgx2T8pmo/edit?usp=sharing).

Reply:
Please justify the change in #5275